index.js

```diff
@@ -1,17 +1,71 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    if (this.length === 0) {
+      this.items.push(item);
+    } else {
+      let inserted = false;
+      for (let i = 0; i < this.length; i++) {
+        if (item < this.items[i]) {
+          this.items.splice(i, 0, item);
+          inserted = true;
+          break;
+        }
+      }
+      if (!inserted) {
+        this.items.push(item);
+      }
```

Comment:
Creo que te has complicado de mas aqui :D
Te pongo una solucion
```suggestion
    this.items.push(item);
    this.items.sort((a, b) => a - b);
    this.length = this.items.length;
```

Y ahora viene cuando te explota el craneo jajaj sabes que devuelve el metodo push (si el metodo push devuelve algo no solo añade cosas al array), te lo dejo para que lo mires a ver como podriamos incluso reducir eso a 2 lineas :D

Reply:
He buscado que el metodo push devuelve la nueva longitud del array... lo unico que se me ocurre para hacerlo en dos lineas es no usar .push y no se hasta que punto estaria bien, se me ha ocurrido esto:

`this.items = this.items.concat(item).sort((a, b) => a - b);
this.length = this.items.length;`

Mi logica detrás de esto es que .push devuelve la longitud, es decir, no puedo usar .sort en la misma linea, asi que uso .concat para concatenar el 'item' que entra, y ahora si, usar .sort para ordenarlo directamente